lib/url.js

```diff
@@ -62,14 +62,35 @@ helpers.urlParse = function(str) {
 };
 
 /**
- * Strip the query string from the give `url`.
+ * Strip the query string from a `url`.
  *
  * @name .stripQuerystring
  * @param {String} `url`
- * @return {String}
+ * @return {String} the url without the queryString
  * @api public
  */
 
 helpers.stripQuerystring = function(url) {
   return url.split('?')[0];
 };
+
+/**
+ * Strip protocol from a `url`.
+ *
+ * Useful for displaying media that may have an 'http' protocol
+ * on secure connections.  Will change 'http://foo.bar to `//foo.bar`
+ *
+ * @name .stripProtocol
+ * @param {String} `str`
+ * @return {String} the url with http protocol stripped
+ * @api public
+ */
+
+helpers.stripProtocol = function (str) {
+  var parsed = url.parse(str);
+  if (parsed.protocol === 'http:') {
```

Comment:
I don't think this test is necessary. It would be good to just strip the protocol altogether since using `//foo.html` tells the browser to use the same protocol that the original page was loaded with. This would then work for `https:`, `http:`, and `file:`.